kde-apps/libkmahjongg/libkmahjongg-24.12.3.recipe

```diff
@@ -0,0 +1,102 @@
+SUMMARY="Mahjongg library based on Qt/KDE Frameworks"
+DESCRIPTION="This directory contains the library used for loading and rendering of \
+Mahjongg tilesets and associated backgrounds, used by KMahjongg and KShisen."
+HOMEPAGE="https://invent.kde.org/games/libkmahjongg"
+COPYRIGHT="2010-2024 KDE Organisation"
+LICENSE="GNU LGPL v2"
+REVISION="1"
+SOURCE_URI="https://download.kde.org/stable/release-service/$portVersion/src/libkmahjongg-$portVersion.tar.xz"
+CHECKSUM_SHA256="db2bdb5b8214f28ad68fdac424bec23fb1862121b729982c8af26b0b0e13f6c6"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	libkmahjongg6$secondaryArchSuffix = $portVersion
+	lib:lib6libKF6KMahjongglib$secondaryArchSuffix = 6.0.241203 compat >= 6
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	lib:libKF6Codecs$secondaryArchSuffix
```

Comment:
Just a nitpick, I tend to bundle the frameworks for Qt and KF into seperate ones from the "other" requirements like putting a `# KF6 `and `# Qt6` line between the blocks, makes reading it all a lot clearer.
Like:
```
	lib:libGL$secondaryArchSuffix
	...
	# KF6
	lib:libKF6ColorScheme$secondaryArchSuffix
	...
	# Qt6
	lib:libQt6Core$secondaryArchSuffix
	...
```